### ISSUE TITLE
Fix my jQuery Offset/scroll animation

### ISSUE THREAD

**Issue by WJthylin** [WJthylin/William#1]
Hello, and a big thanks to the one being abel to fix this problem!

I'm pretty new to jQuery but i've been coding HTML and CSS for a while now. 
My problem is that my jQuery doesn't work, it's probably a simple problem, but i can't find it. The content won't appear on the webpage, it's just a blank page where i want it to appear.

This is probably the wrong way to upload this, but im pretty new on GitHub to, so..

this is the file that i've been working on  (i've taken away alot, i'm working on my website):

<html>
    <head>
    <meta charset="utf-8">

    <title> Aniamtion</title>

    <style type="text/css">
        .content{
     height: 100px;
    background-color: #dbdbdb;
}

.about{
    max-width: 550px;
    margin-top: 40px;
    padding: 2px 0;
    transform-style: preserve-3d;
    transform:perspective(500px);
}

.about h2{
    text-transform: uppercase;
    font-size: 30px;
    font-weight: 400;
    text-align: center;
}

.about h2, .about p {
    transition: all 0.8s ease;
}

.about.hidden h2{
    opacity: 0;
    transform: translate3d(0, 0, 50px) rotateY(-50deg);
}

.about.hidden p{
    opacity: 0;
    transform: translate3d(0, 0, 50px) rotateY(50deg);
}
        
    </style>    
    
    </head>
   
<div class="content"></div>
        
            <div class="about hidden">
                <h2> This is Heading </h2>
                <p> Lorem Ipsum är en utfyllnadstext från tryck- och förlagsindustrin. Lorem ipsum har 
                 varit </p>
            </div>  
        
        
        <div class="content"></div>
  
        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
        <script type="text/javascript">
            
            (function(){
            
                var aboutEl = $('div.about'),
                aboutElOffest = aboutEl.offset().top/2,
                documentEl = $(document);
                
                documentEL.on('scroll', function(){
                    if ( documentEl.scrollTop() > aboutElOffest && aboutEl.hasClass('hidden') ) aboutEl.removeClass('hidden');
                });
                
            
            })();
        </script>
